testing/env_yamls/activate-env.yml

```diff
@@ -0,0 +1,7 @@
+name: activate-env
+
+channels:
+    - sodre
+
+dependencies:
+    - conda_pack_test_lib3
```

Comment:
The package is part of the PR. You might want to copy it to your own Anaconda Cloud channel.